tensorboard/backend/event_processing/plugin_event_multiplexer_test.py

```diff
@@ -295,30 +295,97 @@ def testAddRunMaintainsLoading(self):
     self.assertTrue(x.GetAccumulator('run1').reload_called)
     self.assertTrue(x.GetAccumulator('run2').reload_called)
 
+  def testAddReloadWithMultipleThreads(self):
+    x = event_multiplexer.EventMultiplexer(max_reload_threads=2)
+    x.Reload()
+    x.AddRun('run1')
+    x.AddRun('run2')
+    x.AddRun('run3')
+    self.assertTrue(x.GetAccumulator('run1').reload_called)
+    self.assertTrue(x.GetAccumulator('run2').reload_called)
+    self.assertTrue(x.GetAccumulator('run3').reload_called)
+
 
 class EventMultiplexerWithRealAccumulatorTest(tf.test.TestCase):
 
   def testDeletingDirectoryRemovesRun(self):
     x = event_multiplexer.EventMultiplexer()
     tmpdir = self.get_temp_dir()
-    join = os.path.join
-    run1_dir = join(tmpdir, 'run1')
-    run2_dir = join(tmpdir, 'run2')
-    run3_dir = join(tmpdir, 'run3')
+    self.add3RunsToMultiplexer(tmpdir, x)
+    x.Reload()
 
-    for dirname in [run1_dir, run2_dir, run3_dir]:
-      _AddEvents(dirname)
+    # Delete the directory, then reload.
+    shutil.rmtree(os.path.join(tmpdir, 'run2'))
+    x.Reload()
+    self.assertNotIn('run2', x.Runs().keys())
+
+  def testReloadWithMoreRunsThanThreads(self):
```

Comment:
Hmm, do these tests need to be using a real accumulator?  It seems like mostly we're checking behavior in EventMultiplexer itself so a real accumulator wouldn't be necessary.

Reply:
Indeed! The tests have been moved to the other suite that makes use of mock accumulators.